__pycache__/battleship.py

```diff
@@ -0,0 +1,313 @@
+"""
+Battleship Project
+Name:
+Roll No:
+"""
+
+import battleship_tests as test
+
+project = "Battleship" # don't edit this
+
+### SIMULATION FUNCTIONS ###
+
+from tkinter import *
+import random
+
+EMPTY_UNCLICKED = 1
+SHIP_UNCLICKED = 2
+EMPTY_CLICKED = 3
+SHIP_CLICKED = 4
+
+
+'''
+makeModel(data)
+Parameters: dict mapping strs to values
+Returns: None
+'''
+def makeModel(data):
+    data["rows"]=10
+    data["cols"]=10
+    data["board_size"]=500
+    data["cell_size"]= 5
+    data["num_ships"]= 5
+
+    for i in range (2):
+        grid=emptyGrid(10, 10)
+        for j in range(5):
+           ship=createShip()
+           check=checkShip(grid,ship)
+           if check==True:
+              addShips(grid,1)
+        if i==0:
+            data["userBoard"]=grid
+        else:
+            data["compBoard"]=grid        
+    
+    return data
+
+
+
+'''
+makeView(data, userCanvas, compCanvas)
+Parameters: dict mapping strs to values ; Tkinter canvas ; Tkinter canvas
+Returns: None
+'''
+def makeView(data, userCanvas, compCanvas):
+    grid= test.testGrid()
+    showShips = True
+    for a in range(2):
+       if a==0:
+          #board=data["userBoard"]
+          canvas=userCanvas
+       else:
+          #board=data["compBoard"]
+          canvas=compCanvas
+      
+    drawGrid(data, canvas, grid, showShips)
+       
+
+    return None
+
+
+'''
+keyPressed(data, events)
+Parameters: dict mapping strs to values ; key event object
+Returns: None
+'''
+def keyPressed(data, event):
+    pass
+
+
+'''
+mousePressed(data, event, board)
+Parameters: dict mapping strs to values ; mouse event object ; 2D list of ints
+Returns: None
+'''
+def mousePressed(data, event, board):
+    pass
+
+#### WEEK 1 ####
+
+'''
+emptyGrid(rows, cols)
+Parameters: int ; int
+Returns: 2D list of ints
+'''
+def emptyGrid(rows, cols):
+    return
+
+
+'''
+createShip()
+Parameters: no parameters
+Returns: 2D list of ints
+'''
+def createShip():
+    return
+
+
+'''
+checkShip(grid, ship)
+Parameters: 2D list of ints ; 2D list of ints
+Returns: bool
+'''
+def checkShip(grid, ship):
+    return
+
+
+'''
+addShips(grid, numShips)
+Parameters: 2D list of ints ; int
+Returns: 2D list of ints
+'''
+def addShips(grid, numShips):
+    return
+
+
+'''
+drawGrid(data, canvas, grid, showShips)
+Parameters: dict mapping strs to values ; Tkinter canvas ; 2D list of ints ; bool
+Returns: None
+'''
+def drawGrid(data, canvas, grid, showShips):
+      for i in range(data["rows"]):
+        for j in range(data["cols"]):
+            if grid[i][j]==SHIP_UNCLICKED:
+               canvas.create_rectangle(j*50, i*50, (j+1)*50, (i+1)*50,fill="yellow")
```

Comment:
Why are you hardcoding '50', make use of variables which you have declare above.